website/source/docs/providers/aws/r/iam_access_key.html.markdown

```diff
@@ -55,5 +55,8 @@ The following attributes are exported:
 * `id` - The access key ID.
 * `user` - The IAM user associated with this access key.
 * `secret` - The secret access key. Note that this will be written to the state file.
+* `ses_smtp_password` - The secret access key converted into an SES SMTP
```

Comment:
At this point, shouldn't this be encrypted with PGP in the same way the `secret` is?